canonical-web-and-design/maas-ui
# refactor(ui): move machine storage utils to node storage utils

## Done

- Moved machine storage utils to node storage utils file
- Updated storage device test factory names to be prefixed with "node" instead of "machine" 

## QA

### MAAS deployment

To run this branch you will need access to one of the following MAAS deployments:

- [Karura](/HACKING.md#karura)
- [Bolla](/HACKING.md#bolla)
- [A development MAAS](/HACKING.md#development-deployment)
- [A local snap MAAS](/HACKING.md#snap-deployment) (this will not usually have machines)

### Running the branch

You can run this branch by:

- Serving with [dotrun](/HACKING.md#maas-ui-development-setup)
- [Building in a development MAAS](/HACKING.md#running-maas-ui-from-a-development-maas)

### QA steps

- Nothing should have changed. Check that you can still see machine storage and you can edit storage for a machine in Ready or Allocated state.

## Fixes

Fixes canonical-web-and-design/app-tribe#985
